Survey Checks/README.md

```diff
@@ -0,0 +1,32 @@
+# Survey checks
+
+Here we provide detailed instructions for preparing your electronic survey with the aim of adding survey checks and creating a real-time high-frequency check dashboard.
+
+1. [Prepare Survey forms](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/Survey%20Checks/1-prepare-scto-forms.md)
```

Comment:
it is not very clear now whether this is generally about designing an scto form or specifically about designing one for survey checks. In practice, it seems like the readme is mostly about the adaptations needed to do real-time checks with an existing survey form. 

I wonder if we structure this so that it's somehow clear that the input to iesurveychecks is a programmed SCTO questionnaire. and then start with how you need to adapt that to be able to run these checks.

Reply:
Perhaps on the initial readme, should specify inputs and outputs at each step, so that the flow is clearer.

Reply:
in practice a lot of the content here is about the types of checks and i think should move to main readme. then maybe this step becomes preparing pre-load data.